pkg/daemon/daemon.go

```diff
@@ -1401,10 +1402,8 @@ func (dn *Daemon) prepUpdateFromCluster() (*mcfgv1.MachineConfig, *mcfgv1.Machin
 	return currentConfig, desiredConfig, nil
 }
 
-// completeUpdate marks the node as schedulable again, then deletes the
-// "transient state" file, which signifies that all of those prior steps have
-// been completed.
-func (dn *Daemon) completeUpdate(desiredConfigName string) error {
+// uncordonAndUntaintNode uncordons the node and removes the update in progress taint
+func (dn *Daemon) uncordonAndUntaintNode() error {
```

Comment:
Ok bummer. To explain a little more of why I think it's confusing: on layering right now (which I'm not sure is even correct), when an update finishes (https://github.com/mkenigs/machine-config-operator/blob/demo/pkg/daemon/update.go#L2133) we:
`setNodeAnnotation()`
`completeUpdate()`
`SetDone()`
`Eventf("NodeDone")`
Maybe it's because I'm less familiar with the codebase, but despite the fact that I wrote that sequence of calls I still find myself going to definition for `completeUpdate` to remind myself what it does.

Are you good with dropping the desiredConfigName argument and some of the other changes, or should I just close the PR?

Reply:
I personally prefer not to make this set of changes as I find completeUpdate clearer and part of my mental map of this part of the codebase.